tools/create-chipsec.sh

```diff
@@ -62,7 +62,7 @@ sign_kernel () {
 	sbsign --key "$keypath"/DB.key --cert "$keypath"/DB.crt --output ${KERNEL} ${KERNEL}
 
 	# Also sign the Chipsec module
-	"${mount_point}"/usr/lib/linux-kbuild-4.19/scripts/sign-file \
+	"${mount_point}"/usr/lib/linux-kbuild-*/scripts/sign-file \
```

Comment:
So the reason I changed it is because it broke with recent updates. Fair point about the possibility of several kernels (which I addressed somewhere else in this file). It doesn't seem to happen in practice in debootstrap, but fixed anyway.